api/v1beta1/cluster_types.go

```diff
@@ -418,6 +420,8 @@ type ClusterStatus struct {
 
 	// Conditions defines current service state of the cluster.
 	// +optional
+	// +Metrics:stateset:name="status_condition",help="The condition of a cluster.",labelName="status",JSONPath=.status,list={"True","False","Unknown"},labelsFromPath={"type":".type"}
```

Comment:
You can/should add `nilIsZero: true` for statesets

Reply:
thx!

TODO: also review all other statesets?!